zetaclient/tss/tss_signer.go

```diff
@@ -419,17 +413,24 @@ func (tss *TSS) SignBatch(
 	return sigBytes, nil
 }
 
-// Validate validates the TSS
-func (tss *TSS) Validate() error {
+// ValidateAddresses validates the TSS EVM and BTC addresses
+func (tss *TSS) ValidateAddresses(btcChainIDs []int64) error {
+	log.Info().Msgf("tss.pubkey: %s", tss.CurrentPubkey)
```

Comment:
Consider passing logger to TSS instance i.e. `tss.logger` with appropriate log fields (to allow filtering by tss logs only)

Reply:
added fields https://github.com/zeta-chain/node/pull/2907/commits/3a111e0f83c488e04fec081cc97bb56634b51ada